Permission issues when running `make shell`

when i run `make shell`, i successfully create the dev image and the `.dev-last-build` file, but i'm getting a permission error for the `docker` command. note that i am able to run `docker run hello-world` as well as the `docker` command that is invoked by calling make.

Error:
```bash
$ make shell
docker run --network noteworthy --rm -it -v /usr/local/bin/docker:/usr/local/bin/docker -v /var/run/docker.sock:/var/run/docker.sock -v /home/patrickdlg/src/decentralabs/forks/noteworthy:/opt/noteworthy --name noteworthy-dev-13-59_12-15 noteworthy:dev
make: docker: Permission denied
make: *** [Makefile:44: shell] Error 127
```

running the command myself:
```bash
docker run --network noteworthy --rm -it -v /usr/local/bin/docker:/usr/local/bin/docker -v /var/run/docker.sock:/var/run/docker.sock -v /home/patrickdlg/src/decentralabs/forks/noteworthy:/opt/noteworthy --name noteworthy-dev-13-59_12-15 noteworthy:dev
root@69bc1baa82d0:/opt/noteworthy#
```

not sure if this is an issue with my environment.

--- a/.gitignore
+++ b/.gitignore
@@ -3,3 +3,4 @@ __pycache__
 build/
 dist/
 *.egg-info/
+.dev-last-build

--- a/Makefile
+++ b/Makefile
@@ -19,26 +19,26 @@ help:
 	@echo
 
 test:
-	./run-tests.sh
+	./run-tests.sh ;
 
 integration-test:
-	HUB_IP=$(HUB_IP) ./notectl/run-tests.sh integration
+	HUB_IP=$(HUB_IP) ./notectl/run-tests.sh integration ;
 
 gatling:
-	docker run --network noteworthy --rm -it -v $(ROOT_DIR)/perftest/gatling/user-files:/opt/gatling/user-files -v $(ROOT_DIR)/perftest/gatling/results:/opt/gatling/results --name noteworthy-gatling-$(shell date +"%H-%M_%m-%d") denvazh/gatling -s $(SIM_NAME)
+	docker run --network noteworthy --rm -it -v $(ROOT_DIR)/perftest/gatling/user-files:/opt/gatling/user-files -v $(ROOT_DIR)/perftest/gatling/results:/opt/gatling/results --name noteworthy-gatling-$(shell date +"%H-%M_%m-%d") denvazh/gatling -s $(SIM_NAME) ;
 
 docker:
-	docker build --build-arg GIT_COMMIT=$(GIT_COMMIT) --build-arg RELEASE_TAG=$(RELEASE_TAG) -t decentralabs/noteworthy:$(ROLE)-$(RELEASE_TAG) -f Dockerfile.$(ROLE) .
+	docker build --build-arg GIT_COMMIT=$(GIT_COMMIT) --build-arg RELEASE_TAG=$(RELEASE_TAG) -t decentralabs/noteworthy:$(ROLE)-$(RELEASE_TAG) -f Dockerfile.$(ROLE) . ;
 
 docker-push:
-	echo ${DOCKERHUB_PSW}| docker login -u ${DOCKERHUB_USR} --password-stdin
-	docker tag decentralabs/noteworthy:$(ROLE)-$(RELEASE_TAG) decentralabs/noteworthy:$(ROLE)-$(RELEASE_TAG)-$(GIT_COMMIT)
-	docker push decentralabs/noteworthy:$(ROLE)-$(RELEASE_TAG)
-	docker push decentralabs/noteworthy:$(ROLE)-$(RELEASE_TAG)-$(GIT_COMMIT)
+	echo ${DOCKERHUB_PSW}| docker login -u ${DOCKERHUB_USR} --password-stdin ;
+	docker tag decentralabs/noteworthy:$(ROLE)-$(RELEASE_TAG) decentralabs/noteworthy:$(ROLE)-$(RELEASE_TAG)-$(GIT_COMMIT) ;
+	docker push decentralabs/noteworthy:$(ROLE)-$(RELEASE_TAG) ;
+	docker push decentralabs/noteworthy:$(ROLE)-$(RELEASE_TAG)-$(GIT_COMMIT) ;
 
 .dev-last-build: requirements.base.txt requirements.dev.txt
-	docker build -t noteworthy:dev -f Dockerfile.dev .
-	echo $(shell date) > .dev-last-build
+	docker build -t noteworthy:dev -f Dockerfile.dev . ;
+	echo $(shell date) > .dev-last-build ;
 
 dev shell: .dev-last-build
-	docker run --network noteworthy --rm -it -v /usr/local/bin/docker:/usr/local/bin/docker -v /var/run/docker.sock:/var/run/docker.sock -v $(ROOT_DIR):/opt/noteworthy --name noteworthy-dev-$(shell date +"%H-%M_%m-%d") noteworthy:dev
+	docker run --network noteworthy --rm -it -v /usr/local/bin/docker:/usr/local/bin/docker -v /var/run/docker.sock:/var/run/docker.sock -v $(ROOT_DIR):/opt/noteworthy --name noteworthy-dev-$(shell date +"%H-%M_%m-%d") noteworthy:dev;

